src/lib/app.ts

```diff
@@ -266,12 +266,12 @@ export class App extends events.EventEmitter {
 		.then(() => warning)
 	}
 
-	createDockerfile(options) {
+	createDockerfile(options?) {
 		let dockerfile = path.join(this.path, 'Dockerfile')
 		let dbProd = this.config.get<IDatabaseConfig>(AppMode.PRODUCTION, ConfigType.DATABASE)
 		let webProd = this.config.get<IServerConfig>(AppMode.PRODUCTION, ConfigType.SERVER)
 		fs.writeFileSync(dockerfile, `FROM node:7.10-alpine
-MAINTAINER ${options.author}
+MAINTAINER ${options && options.author ? options.author : 'me@company.com'}
```

Comment:
I already considered the author implementation as a bug in v0.8. It would be nice (maybe not for this PR I don't know) to use the login information from Materia Designer to help determine the author:

e.g. when we log into Materia Designer, we could write in a file (e.g. ~/.materia/user.json) the name, email etc. and then, in materia-server, we could try to read it by default to search author information.

something like:
```ts
try {
    const user = JSON.parse(fs.readFileSync('~/.materia/user.json', 'utf-8'));
} catch (e) { }
fs.writeFileSync(dockerfile, `FROM node:7.10-alpine
MAINTAINER ${options && options.author || user && user.email || 'me@company.com'}
```